File: Bubble_sort/bubbleSort.java
/*In bubble sort we compare the first two elements and then switch them according to the value*/


public class bubbleSort {
    public static void main(String[] args) {
        int list[] = {2, 4, 62, 25, 6, 36, 52};
        int n = list.length;
        
        for (int i = 0; i < n - 1; i++) {
            for (int j = 0; j < n - i - 1; j++) {
                if (list[j] > list[j + 1]) {
                    // Swap list[j] and list[j + 1]
                    int temp = list[j];
                    list[j] = list[j + 1];
                    list[j + 1] = temp;
                }
            }
        }

        for (int i = 0; i < n; i++) {
            System.out.println(list[i]);
        }
    }
}

File: Bubble_sort/bubble_sort.py
def main():
    list = [34,52,41,31,643,24]
    n = len(list)

    for i in range(n-1):
        for j in range(0,n-i-1):
            if list[j] > list[j+1]:
                temp = list[j]
                list[j] = list[j+1]
                list[j+1] = temp

    for i in range(n):
        print(list[i])

main()